<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpScreen</class>
 <widget class="QWidget" name="HelpScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>822</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="tutorialLabel">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QLabel {
    border: 5px solid #87e8b9;
    border-radius: 10px;
    background-color: #cfffe8;
    color: black;
    font: bold 15px;
    padding: 10px; 
    min-width: 40px;
    min-height: 40px;
}
</string>
     </property>
     <property name="text">
      <string>WELCOME TO THE TUTORIAL! CLICK ON THE AND GATE, OR GATE, OR NOT GATE TO BEGIN LEARNING ABOUT LOGIC GATES. 

EXPLORE ALL ABOUT TRUTH TABLES, HOW LOGIC GATES WORK AND APPLICATIONS OF LOGIC GATES IN THE REAL WORLD!</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="homeButton">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
    border: 5px solid #7398ff;
    border-radius: 10px;
    background-color: #a6bdff;
    color: black;
    font: bold 24px;
    padding: 10px; 
    min-width: 40px;
    min-height: 40px;
}

QPushButton:hover {
    background-color: #c7d5ff;
}

QPushButton:pressed {
    background-color: #a3baff;
}</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="logicButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    border: 5px solid #f2ad46;
    border-radius: 10px;
    background-color: #ffdba6;
    color: black;
    font: bold 20px;
    padding: 10px; 
    min-width: 40px;
    min-height: 40px;
}

QPushButton:hover {
    background-color: #fff4cf;
}

QPushButton:pressed {
    background-color: #fab969;
}</string>
       </property>
       <property name="text">
        <string>HOW DO LOGIC 
GATES WORK?</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="notButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    border: 5px solid #ebcf44;
    border-radius: 10px;
    background-color: #fffbad;
    color: black;
    font: bold 24px;
    padding: 10px; 
    min-width: 40px;
    min-height: 40px;
}

QPushButton:hover {
    background-color: #f5f3bf;
}

QPushButton:pressed {
    background-color: #fff27a;
}</string>
       </property>
       <property name="text">
        <string>NOT GATE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="andButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    border: 5px solid #e84a4a;
    border-radius: 10px;
    background-color: #f78686;
    color: black;
    font: bold 24px;
    padding: 10px; 
    min-width: 40px;
    min-height: 40px;
}

QPushButton:hover {
    background-color: #f7b2b2;
}

QPushButton:pressed {
    background-color: #fa9b9b;
}</string>
       </property>
       <property name="text">
        <string>AND GATE</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="truthButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    border: 5px solid #b97de8;
    border-radius: 10px;
    background-color: #e7c9ff;
    color: black;
    font: bold 20px;
    padding: 10px; 
    min-width: 40px;
    min-height: 40px;
}

QPushButton:hover {
    background-color: #fbe8ff;
}

QPushButton:pressed {
    background-color: #eca8ff;
}</string>
       </property>
       <property name="text">
        <string>WHAT IS A
TRUTH TABLE?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="exampleButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    border: 5px solid #61de5b;
    border-radius: 10px;
    background-color: #a5ffa1;
    color: black;
    font: bold 20px;
    padding: 10px; 
    min-width: 40px;
    min-height: 40px;
}

QPushButton:hover {
    background-color: #d6ffd1;
}

QPushButton:pressed {
    background-color: #75d674;
}</string>
       </property>
       <property name="text">
        <string>REAL WORLD
LOGIC GATE
EXAMPLES</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="orButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    border: 5px solid #40cfff;
    border-radius: 10px;
    background-color: #abeaff;
    color: black;
    font: bold 24px;
    padding: 10px; 
    min-width: 40px;
    min-height: 40px;
}

QPushButton:hover {
    background-color: #e8fffd;
}

QPushButton:pressed {
    background-color: #bffff9;
}</string>
       </property>
       <property name="text">
        <string>OR GATE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
